app/gui/view/src/project.rs

```diff
@@ -41,6 +44,9 @@ use ide_view_graph_editor::NodeSource;
 /// Browser when user is quickly typing in the expression input.
 const INPUT_CHANGE_DELAY_MS: i32 = 200;
 
+/// The height for the top bar.
+const TOP_BAR_HEIGHT: f32 = 38.0;
```

Comment:
It was derived from the current state. Since I thought the top bar was carefully designed, I didn't want to mess it up. Do you want me to change it? I think it's 50px in the Figma design.